2_base_and_sensor.ipynb

```diff
@@ -309,48 +575,90 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's compare the state of the simulator (on Gazebo) with the displayed data (on RViz).\n",
```

Comment:
"Let's compare the state of the simulator (in Gazebo) with the displayed data (in RViz).\n",